app-backend/db/src/main/scala/UserDao.scala

```diff
@@ -331,8 +332,50 @@ object UserDao extends Dao[User] with Sanitization {
         Some(userBulkCreate.platformId),
         Some(userBulkCreate.organizationId)
       )
-      _ <- userBulkCreate.campaignId traverse { campaignId =>
+      copiedCampaignO <- userBulkCreate.campaignId traverse { campaignId =>
         CampaignDao.copyCampaign(campaignId, user)
       }
-    } yield user
+    } yield UserWithCampaign(user, copiedCampaignO)
+
+    userBulkCreate.grantAccessToParentCampaignOwner match {
```

Comment:
Is this branching logic tested? I think the expectations that you describe in this PR's testing instructions are encodable in automated tests